base/compiler/abstractinterpretation.jl

```diff
@@ -124,20 +124,17 @@ function abstract_call_gf_by_type(interp::AbstractInterpreter, @nospecialize(f),
         if splitunions
             splitsigs = switchtupleunion(sig)
             for sig_n in splitsigs
-                result = abstract_call_method(interp, method, sig_n, svec(), multiple_matches, sv)
-                rt = result.rt
-                edge = result.edge
+                (; rt, edge, effects) = result = abstract_call_method(
+                    interp, method, sig_n, svec(), multiple_matches, sv)
```

Comment:
Having these all on the same line feels a bit excessive to me.

Reply:
Separated into two lines.